libs/wingsdk/src/std/json.ts

```diff
@@ -146,12 +146,13 @@ export class Json {
   private constructor() {}
 
   /**
-   * Returns a specified element from the Json.
```

Comment:
Do we want to keep the existing implementation and add TryGet()?

Reply:
`Json.tryGet()` and `Json.tryGetAt()` already exist.